### PR TITLE
chore: improve studio

### PR DESCRIPTION
**Pull request by LeoFVO**

### CODE REVIEW

File: apps/polyflix/src/modules/studio/pages/users/[create-update].page.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { useGetUserQuery, useGetUsersQuery } from '@users/services/user.service'
+import { InformationsForm } from '@users/components/Forms/InformationsForm.component'
+
+export const CreateUpdateUserPage = () => {
+  const { t } = useTranslation('users')
+  const { id } = useParams<{ id: string }>()
+
+  const { data: user, isLoading } = useGetUserQuery(id)
+  // const { data, isLoading } = useGetUsersQuery({ page: 1, pageSize: 1 })
+  // const user = data?.data[0]
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/studio/pages/users/[create-update].page.tsx:18]: Remove comment ?

File: apps/polyflix/src/modules/studio/pages/videos/videos-list.page.tsx [hunk 2]
```diff
@@ -73,15 +78,61 @@ export const VideosListPage = () => {
...
+                    marginRight={1}
+                  >
+                    <ImageCover ratio={16 / 9} src={video.thumbnail} />
+                  </Box>
+
+                  <Stack direction="column" spacing={0}>
+                    <Typography variant="body1" display="flex" lineHeight={1}>
+                      {video.title}
+                      <Typography sx={{ fontSize: '0.8em', marginLeft: 1 }}>
+                        {t('common.informations.publish')}{' '}
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/studio/pages/videos/videos-list.page.tsx:105]:
```suggestion
                        {t('common.informations.publish')}
```

File: apps/polyflix/src/modules/studio/pages/videos/videos-list.page.tsx [hunk 3]
```diff
@@ -73,15 +78,61 @@ export const VideosListPage = () => {
...
+                        variant="middle"
+                        sx={{ height: '0.6em' }}
+                        flexItem
+                      />
+                      {
+                        t('slug.details.tooltips.likes', {
+                          ns: 'videos',
+                          count: abbreviateNumber(
+                            video?.likes || 0
+                          ) as unknown as number,
```
Comment (MaloPolese) [inline apps/polyflix/src/modules/studio/pages/videos/videos-list.page.tsx:129]: as unknown ?

**Comment by MaloPolese** [comment]
@LeoFVO 
image

File: apps/polyflix/src/modules/studio/pages/videos/videos-list.page.tsx (hunk 2 above)
Comment (LeoFVO) [reply apps/polyflix/src/modules/studio/pages/videos/videos-list.page.tsx:105]: We need this space between i18n text and user firstname

**Review by MaloPolese — APPROVED** [review]

**Pull request merged**